packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
-## v1.1.0-dev (TBD)
-- Support Storefront Preview 
-  - [#1413](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1413)
-  - [#1440](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1440)
-- Fix checkout card number [#1424](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1424)
```

Comment:
Don't we want to _keep_ Storefront Preview CHANGELOG updates in `develop`? We would keep removing them in release PRs for any releases that go out before Storefront Preview

Reply:
Restored changelogs for storefront preview under 2.1.0-dev